sass/components/rtl_support/_badges.scss

```diff
@@ -0,0 +1,17 @@
+// RTL Badges
+.rtl {
+	span.badge {
+		margin-left: initial;
+		margin-right: 14px;
+		float: left;
+
+		&.new:after {
+			content: " جدید";
```

Comment:
@tomscholz Also in Arabic, and yeah I know, I was meaning to ask why should the arabic text be included in the official materialize library?
@arminmon It should be removed anyways so it isn't in the official library.
P.S: Tom, it's everything rtl (by looking at the code).